app/views/simulation.scala.html

```diff
@@ -0,0 +1,96 @@
+@(script: String => Html, style: String => Html)(implicit request: Request[_])
+
+@route(path: String) = @{
+  routes.Assets.at(path).absoluteURL
+}
+
+<html>
+  <head>
+    @style(route("lib/codemirror/lib/codemirror.css"))
+    @style(route("stylesheets/widgets.css"))
+    @style(route("stylesheets/classic.css"))
+    @style(route("stylesheets/netlogo-syntax.css"))
+    @style(route("stylesheets/spinner.css"))
+  </head>
+  <body style="margin: 0px;">
+    @views.html.spinner()
+    <div id="model-container" style="display: inline-block;"></div>
+  </body>
+  @script(route("lib/jquery/jquery.js"))
```

Comment:
So I finally caved. Play's javascriptRoutes works with jquery by default, and it made the ajax for the standalone request feel less hacky. We could swap it out for a minimal ajax library, or just write our own with the same signature as jquery's if you guys prefer.